index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const db = require("./models");
+
+const app = express();
+
+const PORT = process.env.PORT || 4000;
+
+const corsOptions = {
+  origin: "http://localhost:4000"
```

Comment:
I assume you are wanting to have the port dynamic like above by using `${PORT}`

Reply:
Good catch!